src/components/RelicsTab.jsx

```diff
@@ -434,6 +449,13 @@ export default function RelicsTab() {
               Delete Relic
             </Button>
           </Popconfirm>
+          <Switch
+            style={{ marginTop: 5 }}
+            defaultChecked
+            checkedChildren="selected characters"
+            unCheckedChildren="show all characters"
+            onChange={() => { setFilterState(!filterstate) }}
+          />
```

Comment:
After seeing it I think a toggle is probably not the right component (oops my fault for not thinking about it ahead of time on the ticket), changing it to a Select for readability, and we can potentially add more options to the selector